agent/js/src/wd_server.js

```diff
@@ -1028,13 +1028,34 @@ WebDriverServer.prototype.onTracingMessage_ = function(message) {
 };
 
 /**
- * Runs lighthouse with the options specified by the task and dumps data into 
+ * Runs lighthouse with the options specified by the task and dumps data into
  * a directory.
  * @private
  */
 WebDriverServer.prototype.runLighthouse_ = function() {
-  // TODO(phulce): actually run lighthouse
-  this.onPageLoad_(new Error('Lighthouse not yet implemented!'));
+  this.lighthouseFile_ = path.join(this.runTempDir_, 'lighthouse.html');
+  this.app_.timeout(1000, 'Waiting for Chrome to be available...');
+  process_utils.scheduleExec(this.app_,
+    'lighthouse',

```

Comment:
Yeah the most helpful instructions I actually found were over in the [google sites page](https://sites.google.com/a/webpagetest.org/docs/private-instances/node-js-agent/setup) rather than any README so I figured we'd add it there, but even better to find a place here if there's one I missed!